### ISSUE TITLE
Swordfight running on WinBoard 4.8.0 + Leiningen Installer (W7 32-bit)

### ISSUE THREAD

**Issue by ChessGUI** [evalapply/Swordfight#1]
Hello evalapply,

cf. my thread on http://talkchess.com/forum/viewtopic.php?topic_view=threads&p=710812&t=63621
**Could you please upload two complete packages (separated for Windows 32/64 bit systems)
Java JDK 8 + Leiningen Installer 1.0 + WinBoard GUI 4.8.0 on GitHub please?**
What´s about **Arena Interface 3.5.1**  http://www.playwitharena.com/ that can handle batch-files too?

**Comment by replomancer** [comment]
Hello,
Thank you for feedback!

I will register and reply on the forum today (the project is not mature and I know of some problems).

I have just uploaded a jar file here:
https://github.com/evalapply/Swordfight/releases/download/2017.04.03/swordfight-2017.04.03.jar
It can be run with "java -jar swordfight-2017.04.03.jar". I have only tried it on 64 bit systems but it should work on 32 bits just fine (it's java bytecode without native libraries). Leiningen is not needed to run the jar file.

I've just tried Arena for the first time and the engine works (Engines->Install New Engine... change the file type to "Java (*.jar)", find swordfight-2017.04.03.jar and select WB interface). Leiningen is not necessary.

**Comment by ChessGUI** [comment]
[Event "Computer Chess Game"]
[Site "I5COREQUAD"]
[Date "2017.04.04"]
[Round "1"]
[White "Swordfight"]
[Black "Fairy-Max 4.8V"]
[Result "*"]
[TimeControl "40/300"]
[Annotator "1... +0.18"]

1. Nf3 Nc6 {+0.18/9 5} 2. Nc3 d5 {+0.13/9 10} 3. d3 Nf6 {+0.19/8 4} 4. Bg5
h6 {+0.16/10 11} 5. Rg1 hxg5 {+2.88/11 10} 6. Nxg5 Rxh2 {+2.83/10 5} 7. Nf3
Rh5 {+2.94/9 4} 8. Nd2 Bg4 {+3.10/9 5} 9. f3 Qd6 {+3.07/11 6} 10. e4 Qg3+
{+3.95/9 8} 11. Ke2 Nd4+ {+5.14/9 4} 12. Ke3 e5 {+5.33/9 12} 13. Nxd5 Nxd5+
{+10.40/9 4} 14. exd5 g6 {+79.95/12 8} 0:1

played on WinBoard 4.8.0 and Windows 7 SP1 32-bit; obviously has Swordfighjt general problems with time forfeit -by the way: could you create a jar-file of https://github.com/philleski/tactician too?
It is supporting UCI, but Winboard-Interface, ChessGUI and Arena have ports for both free protocols.
http://www.shredderchess.com/download.html - http://download.shredderchess.com/div/uci.zip

What is your email-address, I could send you the debug files and complete engine´s output.

**Comment by replomancer** [comment]
Yes, the engine currently ignores timeouts completely. It's also rather slow (bitboards are on TODO list, but there are more things I need to do). XBoard/Winboard allows to play the game after timeout by default and on faster machines I don't even reach the timeout, so the timer is low on my priority list.

I tried to create a jar file for tactician, but failed (I'm not at all a Java guy). Hopefully the author will help you (I saw you created an issue).

The email address is lukasz.kozuchowski@gmail.com
